lib/request.js

```diff
@@ -343,7 +343,7 @@ module.exports = {
 
   get idempotent() {
     const methods = ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'];
-    return !!~methods.indexOf(this.method);
```

Comment:
koa need support node 4 and `Array.includes` didn't support in this version.

https://github.com/koajs/koa/blob/master/package.json#L59

Reply:
Okay, THX